xknx/devices/sensor.py

```diff
@@ -28,6 +196,173 @@
 class Sensor(Device):
     """Class for managing a sensor."""
 
+    supported_dpts: tuple[type[DPTBase], ...] = (
```

Comment:
this list can be generated by `[dpt for dpt in DPTNumeric.dpt_class_tree()]`  (+ the DPTString ones)